packages/ui/src/components/ShareProjDialog.tsx

```diff
@@ -153,22 +153,20 @@ function CollaboratorList({ repoId, collaborators, dispatch, isOwner }) {
       {collaborators?.map((collab) => (
         <ListItem
           secondaryAction={
-            isOwner && (
-              <IconButton
-                edge="end"
-                aria-label="delete"
-                onClick={() =>
-                  deleteCollaborator({
-                    variables: {
-                      repoId: repoId,
-                      collaboratorId: collab.id,
-                    },
-                  })
-                }
-              >
-                <CloseIcon />
-              </IconButton>
-            )
+            <IconButton
+              edge="end"
+              aria-label="delete"
+              onClick={() =>
+                deleteCollaborator({
```

Comment:
I don't think non-owner user can delete other collaborators. I know it must give an error message from database update operation because of no auth. But in the front-end, we should still disable it.

Reply:
You're right; the current API-auth will throw errors.

I'm trying to remove user/auth-related code from the `packages/ui` so that it can be better re-used across desktop-app and web-app. Things will break. But I consider this as a non-critical break.

Actually, we likely need to add more refined permission control here, e.g., collaborators should be able to add/remove (the behavior of Google Docs).